ts/screens/wallet/payment/PickPaymentMethodScreen.tsx

```diff
@@ -98,13 +100,25 @@ class PickPaymentMethodScreen extends React.Component<Props> {
 
           <View style={WalletStyles.paddedLR}>
             <View spacer={true} />
-            <H1> {I18n.t("wallet.payWith.title")} </H1>
+            <H1>
+              {I18n.t(
```

Comment:
you should use i18n.js [pluralization feature](https://github.com/fnando/i18n-js#setting-up) (search for _pluralization_)